h2o-hadoop/scripts/docker/common/sbin/download_spark

```diff
@@ -0,0 +1,48 @@
+#! /bin/bash
+
+NC='\033[0m'
+RED='\033[91m'
+
+if [[ ${SPARK_VERSIONS} == '' ]]; then
+  echo "Skipping Spark download"
+  exit 0
+fi
+
+IFS=',' read -r -a array <<< ${SPARK_VERSIONS}
+for spark_version in "${array[@]}"; do
+  case "${spark_version}" in
+  1.6)
```

Comment:
We need way to pass arbitrary Sparkling Water binary.

Reply:
It'll be possible to tell which version of SW should be downloaded or provide custom version